build.sh

```diff
@@ -224,12 +224,12 @@ then
     rm -rf ./'Debug-build'
     mkdir ./'Debug-build'
     cd ./'Debug-build'
-    cmake -D CMAKE_BUILD_TYPE:STRING=Debug ../
+    cmake -D CMAKE_BUILD_TYPE:STRING=Debug -DSVF_SANITIZE="${SVF_SANITIZER}" ../
 else
     rm -rf ./'Release-build'
     mkdir ./'Release-build'
     cd ./'Release-build'
-    cmake -D CMAKE_BUILD_TYPE:STRING=Release -DSVF_ENABLE_ASSERTIONS:BOOL=true ../
+    cmake -D CMAKE_BUILD_TYPE:STRING=Release -DSVF_ENABLE_ASSERTIONS:BOOL=true -DSVF_SANITIZE="${SVF_SANITIZER}" ../
```

Comment:
The sanitizer version should be built with this parameter mainly for CI test. The local run using ‘build.sh’ should be fast clean one.

Reply:
The default build is already without the sanitizer.
The sanitizer is only enabled if the `SVF_SANITIZER` variable is not empty - otherwise it doesn't have any impact and is ignored.

Here for the sanitizer build (https://github.com/SVF-tools/SVF/actions/runs/3555462189/jobs/5972171935#step:6:67)
```
-- Sanitizer build: address
```

And for the standard build (https://github.com/SVF-tools/SVF/actions/runs/3555462189/jobs/5972171849#step:6:54)
there is no output.

Reply:
I see. Thanks